#!/usr/bin/env python2.7
PACKAGE = "ibvs_control"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("ibvs_switch",   bool_t,   0, "ibvs is on if true",  False)

exit(gen.generate(PACKAGE, "ibvs_controller", "IbvsController"))